misc/contest/tests/hostname.json

```diff
@@ -0,0 +1,56 @@
+{
+    "JobName": "Hostname validation",
+    "Runs": 1,
+    "RunInterval": "3s",
+    "Tags": ["lscpi"],
+    "TestDescriptors": [
+        {
+            "TargetManagerName": "TargetList",
+            "TargetManagerAcquireParameters": {
+                "Targets": [
+                    {
+                        "FQDN": "10.3.0.88",
+                        "ID": "12345"
+                    }
+                ]
+            },
+            "TargetManagerReleaseParameters": {
+            },
+            "TestFetcherName": "literal",
+            "TestFetcherFetchParameters": {
+                "TestName": "Literal test",
+                "Steps": [
+                    {
+                        "name": "sshcmd",
+                        "label": "SSH TEST",
+                        "parameters": {
+                                "user": ["root"],
+                                "host": ["{{ .FQDN }}"],
+                                "password": ["0penBmc"],
```

Comment:
Might be good to set the default password as a parameter

Reply:
Yes, we will read it from the Yaml file. It will be integrated with the next phase.